### PR TITLE
Rewrite strand artifact docs

### PR DESCRIPTION
**Pull request by takutosato**
@davidbenjamin this is the re-write of the strand artifact docs we've discussed. This model is here to stay until I find time to come up with a new one that addresses the high depth, low allele fraction cases that @madduran pointed out

### CODE REVIEW

**Comment by davidbenjamin** [comment]
Back to @takutosato with comments.

File: docs/mutect/mutect.tex [hunk 1]
```diff
@@ -157,66 +157,58 @@ \section{Somatic Likelihoods Model}\label{introduction}
...
-	\begin{equation}
-	x^+ | f, \epsilon, z \sim
-		\begin{cases}
-			\text{Bin} (n^+, f + \epsilon(1-f)) & z = \mathrm{Art+}\\
-			 \text{Bin} (n^+, f) 			& z = \mathrm{Art-} \\
-			 \text{Bin} (n^+, f)			& z = \mathrm{noArt}
-		\end{cases}
-	\end{equation}
-\end{itemize}
+The strand artifact filter detects sequencing artifact in which the evidence for the alt allele consists entirely of forward reads alone or reverse reads alone. We must detect this while taking into account the fact that at some loci, such as near the end of an exome target, \emph{all} reads are biased towards one direction, and thus the bias towards one read direction among the alt is not suspect.
```
[the remark this answers is not in the document]
Comment (takutosato) [reply docs/mutect/mutect.tex]: done

File: docs/mutect/mutect.tex [hunk 2]
```diff
@@ -157,66 +157,58 @@ \section{Somatic Likelihoods Model}\label{introduction}
 We now have the model evidence for allele subset $\mathbb{A}$.  This lets us choose which alleles are true somatic variants.  It also lets us make calls on somatic loss of heterozygosity events.  Furthermore, instead of reporting max-likelihood allele fractions as before, we may emit the parameters of the Dirichlet posterior $q(\vf)$, which encode both the maximum likelihood allele fractions and their uncertainty.
 
 \section{Strand Artifact Model}
+
 \begin{figure}
 \centering
 \includegraphics[width=0.3\textwidth]{strand_artifact_pgm.png}
-\caption{\label{fig:frog}The probabilistic graphical model}
+\caption{\label{fig:strand artifact}The probabilistic graphical model for the Strand Artifact model}
```
[the remark this answers is not in the document]
Comment (takutosato) [reply docs/mutect/mutect.tex]: done

File: docs/mutect/mutect.tex [hunk 3]
```diff
@@ -157,66 +157,58 @@ \section{Somatic Likelihoods Model}\label{introduction}
...
+\text{Bin} (x^+ | n^+, f) 			& z = \text{Artifact Reverse or No Artifact}\\
+\end{cases}
+\end{equation}
 
-We compute the conditional distributions of $x^-$  analogously.
+The conditional distributions of $x^-$ is defined analogously.
 
-Having observed the read counts in the forward and reverse directions, we can compute the posterior probabilities of the latent variable $z$.  Below we derive the unnormalized posterior probability of strand artifact in forward reads ($z = art+$), given that we observed $x^+$ forward alt reads and $x^-$ reverse alt reads. We use a shorthand $z_0$ to denote $z=art+$ for conciseness. 
+In order to assess whether a variant should be filtered, we compute the posterior probabilities of the artifact states $z$ given the observed alt read counts $X = \{ x^+, x^- \}$.  We compute the likelihood function $P(X | z)$ by marginalizing the latent variables $f$ and $\epsilon$ out of the joint distribution $P(X, f, \epsilon | z)$, and we do this for each of the three possible values of $z$ separately. First consider the case where $z = \text{Artifact Forward}$, which we denote by the shorthand $z^+$ below. The likelihood $P(X | z^+ )$ is
```
[the remark this answers is not in the document]
Comment (takutosato) [reply docs/mutect/mutect.tex]: I agree, done

File: docs/mutect/mutect.tex [hunk 4]
```diff
@@ -157,66 +157,58 @@ \section{Somatic Likelihoods Model}\label{introduction}
...
-			 \text{Bin} (n^+, f)			& z = \mathrm{noArt}
-		\end{cases}
-	\end{equation}
-\end{itemize}
+The strand artifact filter detects sequencing artifact in which the evidence for the alt allele consists entirely of forward reads alone or reverse reads alone. We must detect this while taking into account the fact that at some loci, such as near the end of an exome target, \emph{all} reads are biased towards one direction, and thus the bias towards one read direction among the alt is not suspect.
 
+Let $z$ be a discrete latent random variable that represents the artifact state of a suspected variant, where $z \in \{ \text{Artifact Forward}, \text{Artifact Reverse}, \text{No Artifact} \}$. At each locus, the distributions over forward alt reads $x^+$ and reverse alt reads $x^-$, conditional on the artifact state $z$ as well as the allele fraction $f$ and artifact allele fraction $\epsilon$, are binomial random variables . Artifactual allele fraction $\epsilon$ is the probability that we mistakenly read a ref allele as alt due to a strand artifact error. We give the flat prior over $f$ and a beta prior over $\epsilon$. We define the conditional probability of $x^+$ as
```
[the remark this answers is not in the document]
Comment (takutosato) [reply docs/mutect/mutect.tex]: Done

File: docs/mutect/mutect.tex (hunk 1 above)
[the remark this answers is not in the document]
Comment (takutosato) [reply docs/mutect/mutect.tex]: This must change. Let me get back to you.

[the remark this answers is not in the document]
Comment (davidbenjamin) [reply docs/mutect/mutect.tex]: You can file an issue for it.  Doesn't need to be handled in this PR.

File: docs/mutect/mutect.tex (hunk 4 above)
[the remark this answers is not in the document]
Comment (takutosato) [reply docs/mutect/mutect.tex]: Done

File: docs/mutect/mutect.tex [hunk 5]
```diff
@@ -157,66 +157,58 @@ \section{Somatic Likelihoods Model}\label{introduction}
...
+In order to assess whether a variant should be filtered, we compute the posterior probabilities of the artifact states $z$ given the observed alt read counts $X = \{ x^+, x^- \}$.  We compute the likelihood function $P(X | z)$ by marginalizing the latent variables $f$ and $\epsilon$ out of the joint distribution $P(X, f, \epsilon | z)$, and we do this for each of the three possible values of $z$ separately. First consider the case where $z = \text{Artifact Forward}$, which we denote by the shorthand $z^+$ below. The likelihood $P(X | z^+ )$ is
 
-First we will derive the likelihood $p(x^+, x^-, f, \epsilon | z_0)$
 \begin{align}
-p(x^+, x^- | z_0)  &= \iint_{f, \epsilon}  p(x^+, x^-, f, \epsilon | z_0) \,df\,d\epsilon \\
-			  &= \iint_{f, \epsilon}  p(f) p(\epsilon) p(x^+, x^- | z_0, f, \epsilon) \,df\,d\epsilon \\
-			  &= \iint_{f, \epsilon}  p(f) p(\epsilon) p(x^+ | z_0, f, \epsilon) p(x^- | z_0, f, \epsilon) \,df\,d\epsilon \\
-			  &= \iint_{f, \epsilon}  p(\epsilon) p(x^+ | z_0, f, \epsilon) p(x^- | z_0, f, \epsilon) \,df\,d\epsilon \label{watershed} \\
+P(X | z^+ )  &= \iint_{f, \epsilon}  P(X, f, \epsilon | z^+) \,df\,d\epsilon \nonumber \\
```
[the remark this answers is not in the document]
Comment (takutosato) [reply docs/mutect/mutect.tex]: done

File: docs/mutect/mutect.tex (hunk 1 above)
[the remark this answers is not in the document]
Comment (takutosato) [reply docs/mutect/mutect.tex]: done

File: docs/mutect/mutect.tex [hunk 6]
```diff
@@ -157,66 +157,58 @@ \section{Somatic Likelihoods Model}\label{introduction}
...
-		\end{cases}
-	\end{equation}
-\end{itemize}
+The strand artifact filter detects sequencing artifact in which the evidence for the alt allele consists entirely of forward reads alone or reverse reads alone. We must detect this while taking into account the fact that at some loci, such as near the end of an exome target, \emph{all} reads are biased towards one direction, and thus the bias towards one read direction among the alt is not suspect.
 
+Let $z$ be a discrete latent random variable that represents the artifact state of a suspected variant, where $z \in \{ \text{Artifact Forward}, \text{Artifact Reverse}, \text{No Artifact} \}$. At each locus, the distributions over forward alt reads $x^+$ and reverse alt reads $x^-$, conditional on the artifact state $z$ as well as the allele fraction $f$ and artifact allele fraction $\epsilon$, are binomial random variables . Artifactual allele fraction $\epsilon$ is the probability that we mistakenly read a ref allele as alt due to a strand artifact error. We give the flat prior over $f$ and a beta prior over $\epsilon$. We define the conditional probability of $x^+$ as
 
+\begin{equation}
```
[the remark this answers is not in the document]
Comment (takutosato) [reply docs/mutect/mutect.tex]: I adopted the first format. The second one is really dense - I think it's clearer when we separate the + and -.

File: docs/mutect/mutect.tex (hunk 4 above)
[the remark this answers is not in the document]
Comment (takutosato) [reply docs/mutect/mutect.tex]: done

**Comment by takutosato** [comment]
@davidbenjamin thank you so much for the comments. Back to you.

File: docs/mutect/mutect.tex [hunk 7]
```diff
@@ -161,45 +161,41 @@ \section{Strand Artifact Model}
...
 
-Let $z$ be a discrete latent random variable that represents the artifact state of a suspected variant, where $z \in \{ \text{Artifact Forward}, \text{Artifact Reverse}, \text{No Artifact} \}$. At each locus, the distributions over forward alt reads $x^+$ and reverse alt reads $x^-$, conditional on the artifact state $z$ as well as the allele fraction $f$ and artifact allele fraction $\epsilon$, are binomial random variables . Artifactual allele fraction $\epsilon$ is the probability that we mistakenly read a ref allele as alt due to a strand artifact error. We give the flat prior over $f$ and a beta prior over $\epsilon$. We define the conditional probability of $x^+$ as
+Let $z \in \{ z_+, z_-, z_o \}$ be a latent random variable with 1-hot encoding that represents the artifact state of a suspected variant. $z_+ = 1$ when the candidate variant is in fact a forward strand artifact, $z_- = 1$ when it's a reverse artifact, and $z_o = 1$ when there's no artifacts. At each locus, the conditional distribution over the number of forward alt reads $x^+$ is a binomial random variable
```
Comment (davidbenjamin) [inline docs/mutect/mutect.tex]: You don't need the "in fact."  "There's no artifacts" --> "there is no artifact"

**Pull request merged**